main.js

```diff
@@ -0,0 +1,81 @@
+const apiKey = "0dc56170d9065926c36630f0dff6ee44"
+
+function requestWeatherData(city) {
+  var weatherUrl = "https://api.openweathermap.org/data/2.5/weather?q=" + city + "&units=imperial&appid=" + apiKey;

```

Comment:
would recommend using let/const to declare variables and to stay consistent in your code